spec/clamp/completer_spec.rb

```diff
@@ -1,5 +1,124 @@
+require 'clamp'
+
 RSpec.describe Clamp::Completer do
+  include OutputCapture
+  extend CommandFactory
+
   it "has a version number" do
     expect(Clamp::Completer::VERSION).not_to be nil
   end
+
+  given_command("cmd") do
+    subcommand "complete", "shell autocompletions", Clamp::Completer.new(self)
+    subcommand "render-profile", "Command with hyphen" do
+      def execute
+        puts "Command with hyphen"
+      end
+    end
+  end
+
+  it "generates zsh autocompletion" do
+    command.run(%w[complete zsh])
+    # expect(stdout).to be("render-profile")
+    expect(stdout).to eq(%{#compdef rspec
```

Comment:
Won't this test instantly break whenever we make any change, even eg to whitespace?

Reply:
Yeah that's true. I think one such test in the repo is alright. I would have also like to actually test the autocompletion but thought I might do that as part of another PR.